docs/photoniq/vwrs/04-queue-types.md

```diff
@@ -0,0 +1,24 @@
+---
+sidebar_position: 25
+title: Understanding VWRs Queues
+---
+
+This page describes the different types of queues supported by Virtual Waiting Rooms.
```

Comment:
Do we have to say "this page describes"?

Maybe?
"The different types of queues supported by Virtual Waiting Rooms are outlined." or something like that.